### DISCUSSION TITLE
Email contact doesn't work

### DISCUSSION THREAD

**Discussion by bghjmn32** [lxndrblz/anatole#213] [Q&A]
Hello, I have used your Blog template for a month, everything is perfect and I love it, thank you again.

recently, one of my reader tell me that I they can not use the contact page to send Email to me but the "Email Button" works well (the one always on the left).

the error code is 405 not allowed, I checked the "contact.md" in cotent folder, it's basically empty. Do I need to set something here?

Thank you in advance.

**Comment by lxndrblz** [comment]
Hi @bghjmn32 I've seen you didn't define the Formspree URL for the contact form:

https://github.com/bghjmn32/bghjmn32.github.io/blob/main/config/_default/params.toml

Is that accidental?

**Comment by OriginalMHV** [comment]
I got the same issue with error 405. I've used the endpoint that I got from formspree, declared it inside the _config.toml_ and the Contact page is _True_.
<img width="374" alt="Screenshot 2021-06-28 at 03 01 50" src="https://user-images.githubusercontent.com/60610137/123565448-3a30ce80-d7bd-11eb-814f-85703709a418.png">

**Reply by OriginalMHV** [reply]
Found the problem. It should be inside _params.toml_ instead of _config.toml_. I also found that favicons don't work because of the browser does not update them regularly. It works if you delete the history/data or use another browser.
Thanks for the awesome website :)
